### DISCUSSION TITLE
Welcome to jasmin_smsc_gui Discussions!

### DISCUSSION THREAD

**Discussion by Eudorajab1** [Eudorajab1/jasmin_smsc_gui#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Eudorajab1** [comment]
Please feel free to list any ideas or comments

**Reply by zializa** [reply]
Hi sir may i get this solution be installed

**Reply by Eudorajab1** [reply]
Sorry not sure what the question is

**Reply by zializa** [reply]
Sir I mean - can jasmin and your solution (Web Gui) can be installed in my ubuntu server, as I am not an expert to do it. Can any installation support be provided on payment. If possible let me know.

**Comment by Eudorajab1** [comment]
Hi sure .. if you like I can install all for you on your server for small
fee of €1500,00. I also have a full sms gateway platform which I sell
commercially if you are interested which links to Jasmin and allows you to
do things like clinet billing etc.

Please let me know if you are interested and we can get moving,

Best Regards
John

**Reply by dragelecpc** [reply]
Hello,
Have you still do this? Can i access the demo sir.
I installed your repo and work well, but it lack of billing and history (cdr). So it's very helpful if i can paid you to install the full platform sir.
thank you

**Comment by zializa** [comment]
Thank you very much for your reply, i have a complete solution along with Android Support which does not have SMPP Server. But client is there. So I can connect with SMPP and Http servers.
But my clients who wants to send me via smpp can not connect me via SMPP. Jasmin with Gui interface i can allow my clients to connect me via SMPP and Jasmin can connect my oroginal software via http so that I do not need any additional billing or anything.
And from fiverr or any other site I can get the Jasmine installation done in for just $150
However, thanks for your offer. 
--
Faisal Mohammad Ziaur Rahman 
President & CEO
Prime Line Biz LLC Sunday, 22 January 2023, 09:12PM +04:00 from John Bannister  ***@***.*** :

>Hi sure .. if you like I can install all for you on your server for small
>fee of €1500,00. I also have a full sms gateway platform which I sell
>commercially if you are interested which links to Jasmin and allows you to
>do things like clinet billing etc.
>
>Please let me know if you are interested and we can get moving,
>
>Best Regards
>John
>

**Comment by Eudorajab1** [comment]
No worries ... I have also repaired many such installations so please let
me know if you require any further assistance as I will be glad to help.

Best Regards
John Bannister

**Comment by zializa** [comment]
Thanks a lot bro.
173.233.73.5
user:Elias
pass:Telco1988
Jasmin user and pass
This is also root Password.
If possible can you help us making it up and running.

--
Faisal Mohammad Ziaur Rahman 
President & CEO
Prime Line Biz LLC Sunday, 22 January 2023, 09:40PM +04:00 from John Bannister  ***@***.*** :

>No worries ... I have also repaired many such installations so please let
>me know if you require any further assistance as I will be glad to help.
>
>Best Regards
>John Bannister
>

**Comment by Eudorajab1** [comment]
Sorry I don't understand what it is you are asking me to do. From your last
email you indicated that you would get it all installed for €150 from
somewhere so I really dont understand what it is that you are asking. If
you are asking me to install the jasmin sms_gui for you on your server we
will need to have some agreement for my time to do this.

Best Regards
John Bannister

**Comment by Eudorajab1** [comment]
Hello,

Please can you tell me exactly what it is that you want installing.

Best Regards
John

**Reply by dragelecpc** [reply]
Hi, i need platform like jasmin with more feature like: cdr (history), client portal, mange vender, billing,rates.

**Reply by dragelecpc** [reply]
can you share whatsapp or skype for more discussion sir?

**Comment by Eudorajab1** [comment]
Please send me your whatsapp or skype and I will contact you

**Reply by dragelecpc** [reply]
whatsapp sent to email eudorajab1@gmail.com. Kindly help to check it sir
